src/app/App.js

```diff
@@ -0,0 +1,13 @@
+import MainPage from '../components/mainPage/MainPage';
+import Footer from '../components/footer/Footer';
+
+function App() {
+	return (
```

Comment:
Попробуй настроить IDE, чтобы было не два таба, а 2 пробела

```suggestion
  return (
```